hack/minikube-start.sh

```diff
@@ -16,7 +16,8 @@ minikube start \
     --extra-config=apiserver.service-account-issuer=api \
     --extra-config=apiserver.api-audiences=api,spire-server \
     --extra-config=apiserver.authorization-mode=Node,RBAC \
-    --memory='4096m' \
+    --memory='11264m' \
```

Comment:
Increased the limits to make the build faster.

We can also configure it from env vars later maybe.